internal/operators/manager_test.go

```diff
@@ -0,0 +1,173 @@
+package operators_test
+
+import (
+	"encoding/json"
+
+	"github.com/onsi/ginkgo/extensions/table"
```

Comment:
please group imports, not need to have so many groups

Reply:
:+1: